### ISSUE TITLE
Mobilenet problem 

### ISSUE THREAD

**Issue by davidbp** [plaidml/plaidml#103]
Hi,

I have tried a script that worked on standard keras (OSX) but when I use plaidml.keras it gives me an error. All I have done is to define:

```python
import plaidml.keras

import plaidml.keras
plaidml.keras.install_backend()
from keras import applications
base_model = applications.MobileNet(include_top=False, 
                                          weights='imagenet',
                                          input_tensor=None, 
                                          input_shape=(224, 224, 3), 
                                          pooling=None,
                                          classes=1000)
```

but I get the following error

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-32-6755021c5807> in <module>()
      4                                           input_shape=(224, 224, 3),
      5                                           pooling=None,
----> 6                                           classes=1000)

~/anaconda3/lib/python3.6/site-packages/keras/applications/mobilenet.py in MobileNet(input_shape, alpha, depth_multiplier, dropout, include_top, weights, input_tensor, pooling, classes)
    248             img_input = input_tensor
    249 
--> 250     x = _conv_block(img_input, 32, alpha, strides=(2, 2))
    251     x = _depthwise_conv_block(x, 64, alpha, depth_multiplier, block_id=1)
    252 

~/anaconda3/lib/python3.6/site-packages/keras/applications/mobilenet.py in _conv_block(inputs, filters, alpha, kernel, strides)
    389                strides=strides,
    390                name='conv1')(x)
--> 391     x = BatchNormalization(axis=channel_axis, name='conv1_bn')(x)
    392     return Activation(relu6, name='conv1_relu')(x)
    393 

~/anaconda3/lib/python3.6/site-packages/keras/engine/topology.py in __call__(self, inputs, **kwargs)
    617 
    618             # Actually call the layer, collecting output(s), mask(s), and shape(s).
--> 619             output = self.call(inputs, **kwargs)
    620             output_mask = self.compute_mask(inputs, previous_mask)
    621 

~/anaconda3/lib/python3.6/site-packages/keras/layers/normalization.py in call(self, inputs, training)
    183         if K.backend() != 'cntk':
    184             sample_size = K.prod([K.shape(inputs)[axis]
--> 185                                   for axis in reduction_axes])
    186             sample_size = K.cast(sample_size, dtype=K.dtype(inputs))
    187 

~/anaconda3/lib/python3.6/site-packages/plaidml/keras/backend.py in prod(value, axis, keepdims)
   1180 def prod(value, axis=None, keepdims=False):
   1181     return op.prod(
-> 1182         value, axes=axis, keepdims=keepdims, floatx=ptile.NUMPY_DTYPE_TO_PLAIDML[floatx()])
   1183 
   1184 

~/anaconda3/lib/python3.6/site-packages/plaidml/op.py in prod(value, axes, keepdims, floatx)
   1288 
   1289 def prod(value, axes=None, keepdims=False, floatx=plaidml.DType.FLOAT32):
-> 1290     if not value.shape.ndims:
   1291         return value
   1292     if isinstance(axes, (tuple, list)) and not len(axes):

AttributeError: 'list' object has no attribute 'shape'
```

Anyone knows where the problem might come from?

Before the error I get a message:
```
INFO:plaidml:Opening device "amd_radeon_pro_580_compute_engine.0"
```
suggesting that I am using opencl.

The "same" script in keras ('2.1.5') works fine. That is
```python
import keras

from keras import applications
base_model = applications.MobileNet(include_top=False, 
                                          weights='imagenet',
                                          input_tensor=None, 
                                          input_shape=(224, 224, 3), 
                                          pooling=None,
                                          classes=1000)
```

**Comment by tzerrell** [comment]
This looks like the same problem as #102. We hope to be able to release a fix for this soon.

**Comment by davidbp** [comment]
I have seen you referenced the issue. Is it solved? Please tell me when it is solved and I will test it and let you know. Thank you

**Comment by tzerrell** [comment]
Hi @davidbp, we've finished our own testing and this appears to be fixed, so we've released it as part of 0.3.2. Let us know if it works for you (and reopen if it doesn't)!
